### PR TITLE
Feature/make login and register forms consistent

### PR DESCRIPTION
**Pull request by Tretmax**
image

image

### CODE REVIEW

File: src/containers/LoginForm.tsx [hunk 1]
```diff
@@ -73,14 +79,13 @@ const LoginForm = () => {
...
         ) : null}
       </ToastContainer>
-      <header className="d-flex justify-content-center align-items-center">
-        <Twitter color="blue" size={25} />
-      </header>
 
       <section className="w-50 mt-4 m-auto d-grid gap-4">
-        <h1 className="text-nowrap fs-1 fw-700 ">{t("login.loginTitle")}</h1>
+        <h1 className="text-center text-nowrap fs-2 fw-bold">
+          {t("login.loginTitle")}
```
Comment (ArtemLigerko) [inline src/containers/LoginForm.tsx:68]: Change title from Twitter to Leleka

File: src/containers/LoginForm.tsx [hunk 2]
```diff
@@ -24,8 +25,7 @@ interface MyForm {
 }
 
 const StyledContainer = styled(Container)`
```
Comment (DavidKorystov) [inline src/containers/LoginForm.tsx]: remove this styledContainer, use default Container

**Review by DavidKorystov — CHANGES_REQUESTED** [review]

**Review by DavidKorystov — CHANGES_REQUESTED** [review]
Add validation to the password field and error display in the login component. Like in the username

File: src/containers/Registration.tsx [hunk 3]
```diff
@@ -96,110 +96,126 @@ const Registration = () => {
...
-    </Form>
+    <section className="w-50 mt-4 m-auto d-grid gap-4">
+      <Form className="d-grid" onSubmit={handleSubmit(submitForm)}>
+        <StyledFormFloating className="form-floating">
+          <Form.Control
+            {...register("username")}
+            className="form-control"
+            type="text"
+            name="username"
+            placeholder="Ім'я"
```
Comment (VitaliiDenysiuk) [inline src/containers/Registration.tsx]: use i18n

File: src/containers/Registration.tsx [hunk 4]
```diff
@@ -96,110 +96,126 @@ const Registration = () => {
...
+            errors.username.message &&
+            t(`${errors.username.message}`)}
+        </p>
+        <StyledFormFloating className="form-floating mt-3">
+          <Form.Control
+            className="form-control"
+            {...register("email")}
+            type="text"
+            name="email"
+            placeholder="Ел. пошта"
```
Comment (VitaliiDenysiuk) [inline src/containers/Registration.tsx]: use i18n

File: src/containers/Registration.tsx [hunk 5]
```diff
@@ -96,110 +96,126 @@ const Registration = () => {
...
+            errors.dateOfBirth.message &&
+            t(`${errors.dateOfBirth.message}`)}
+        </p>
+        <StyledFormFloating className="form-floating mt-3">
+          <Form.Control
+            className="form-control"
+            {...register("password")}
+            type="password"
+            name="password"
+            placeholder="Пароль"
```
Comment (VitaliiDenysiuk) [inline src/containers/Registration.tsx]: use i18n

File: src/containers/Registration.tsx [hunk 6]
```diff
@@ -96,110 +96,126 @@ const Registration = () => {
...
+            errors.password.message &&
+            t(`${errors.password.message}`)}
+        </p>
+        <StyledFormFloating className="form-floating mt-3">
+          <Form.Control
+            className="form-control"
+            {...register("confirmPassword")}
+            type="password"
+            name="confirmPassword"
+            placeholder="Підтвердіть пароль"
```
Comment (VitaliiDenysiuk) [inline src/containers/Registration.tsx]: use i18n

**Review by VitaliiDenysiuk — CHANGES_REQUESTED** [review]

File: src/containers/LoginForm.tsx [hunk 7]
```diff
@@ -3,17 +3,17 @@ import {
   Container,
   Form,
   Button,
-  ToastContainer,
-  Toast,
+  // ToastContainer,
+  // Toast,
```
Comment (VitaliiDenysiuk) [inline src/containers/LoginForm.tsx]:
```suggestion

```
remove that

File: src/containers/LoginForm.tsx [hunk 8]
```diff
@@ -119,35 +119,32 @@ const LoginForm = () => {
...
-          <div className="form-floating mb-3">
+          <div className="form-floating mt-3">
             <input
               {...register("password")}
               type="password"
               name="password"
               className="form-control"
               id="floatingInputLoginPassword"
-              placeholder="Name"
+              placeholder="Password"
```
Comment (VitaliiDenysiuk) [inline src/containers/LoginForm.tsx]: use i18n

File: src/containers/LoginForm.tsx [hunk 9]
```diff
@@ -166,14 +163,16 @@ const LoginForm = () => {
           </div>
```
Comment (VitaliiDenysiuk) [inline src/containers/LoginForm.tsx:159]: <Link className="ms-1 text-decoration-none" to="/SignUpModal">
              {t("login.signUp")}
            </Link>
Link did not work
image

File: src/containers/Registration.tsx [hunk 10]
```diff
@@ -96,110 +96,126 @@ const Registration = () => {
...
+        </StyledFormFloating>
+        <p className="text-danger fst-italic mt-1 fs-6">
+          {errors.email && errors.email.message && t(`${errors.email.message}`)}
+        </p>
+        <StyledFormFloating className="form-floating mt-3">
+          <Form.Control
+            className="fs-6"
+            {...register("dateOfBirth")}
+            type="date"
+            name="dateOfBirth"
```
Comment (VitaliiDenysiuk) [inline src/containers/Registration.tsx:157]: at english have Ukrainian placeholder 
image

**Review by VitaliiDenysiuk — CHANGES_REQUESTED** [review]

File: src/containers/LoginForm.tsx (hunk 9 above)
Comment (DavidKorystov) [reply src/containers/LoginForm.tsx:159]: use another link 
image

File: src/containers/Registration.tsx [hunk 11]
```diff
@@ -96,110 +96,126 @@ const Registration = () => {
...
+            htmlFor="floatingInputCustom"
+          >
+            {watchusername ? watchusername.length : 0} / 50
+          </StyledLabel>
+        </StyledFormFloating>
+        <p className="text-danger fst-italic fs-6 mt-1">
+          {errors.username &&
+            errors.username.message &&
+            t(`${errors.username.message}`)}
+        </p>
```
Comment (DavidKorystov) [inline src/containers/Registration.tsx]: Use ternary operators in the error display.
Add i18 to the registration shema, like the login shema.
```suggestion
         {errors.username && (
          <p className="text-danger fst-italic fs-6 mt-1">
            {errors?.username?.message}
          </p>
        )}
```

File: src/containers/Registration.tsx [hunk 12]
```diff
@@ -96,110 +96,126 @@ const Registration = () => {
...
+          <StyledLabel
+            className="pt-1 text-muted"
+            htmlFor="floatingInputCustom"
+          >
+            {t("validation:fields.email")}
+          </StyledLabel>
+        </StyledFormFloating>
+        <p className="text-danger fst-italic mt-1 fs-6">
+          {errors.email && errors.email.message && t(`${errors.email.message}`)}
+        </p>
```
Comment (DavidKorystov) [inline src/containers/Registration.tsx]: Use ternary operators in the error display.
Add i18 to the registration shema, like the login shema.

File: src/containers/Registration.tsx [hunk 13]
```diff
@@ -96,110 +96,126 @@ const Registration = () => {
...
+            htmlFor="floatingInputCustom"
+          >
+            {t("validation:fields.dateOfBirth")}
+          </StyledLabel>
+        </StyledFormFloating>
+        <p className="text-danger fst-italic mt-1 fs-6">
+          {errors.dateOfBirth &&
+            errors.dateOfBirth.message &&
+            t(`${errors.dateOfBirth.message}`)}
+        </p>
```
Comment (DavidKorystov) [inline src/containers/Registration.tsx]: Use ternary operators in the error display.
Add i18 to the registration shema, like the login shema.

File: src/containers/Registration.tsx [hunk 14]
```diff
@@ -96,110 +96,126 @@ const Registration = () => {
...
+            htmlFor="floatingInputCustom"
+          >
+            {t("validation:fields.password")}
+          </StyledLabel>
+        </StyledFormFloating>
+        <p className="text-danger fst-italic mt-1 fs-6">
+          {errors.password &&
+            errors.password.message &&
+            t(`${errors.password.message}`)}
+        </p>
```
Comment (DavidKorystov) [inline src/containers/Registration.tsx]: Use ternary operators in the error display.
Add i18 to the registration shema, like the login shema.

File: src/containers/Registration.tsx [hunk 15]
```diff
@@ -96,110 +96,126 @@ const Registration = () => {
...
+            htmlFor="floatingInputCustom"
+          >
+            {t("validation:fields.confirmPassword")}
+          </StyledLabel>
+        </StyledFormFloating>
+        <p className="text-danger fst-italic mt-1 fs-6">
+          {errors.confirmPassword &&
+            errors.confirmPassword.message &&
+            t(`${errors.confirmPassword.message}`)}
+        </p>
```
Comment (DavidKorystov) [inline src/containers/Registration.tsx]: Use ternary operators in the error display.
Add i18 to the registration shema, like the login shema.

**Review by DavidKorystov — CHANGES_REQUESTED** [review]

File: src/containers/Registration.tsx (hunk 10 above)
Comment (alexandrtovmach) [reply src/containers/Registration.tsx:157]: it's coming from your browser settings I believe

File: src/containers/LoginForm.tsx [hunk 16]
```diff
@@ -23,18 +23,13 @@ interface MyForm {
...
 }
 
-const StyledContainer = styled(Container)`
-  height: 650px;
-  width: 600px;
-`;
-
 const LoginForm = () => {
   const dispatch = useDispatch<AppDispatch>();
   const loginError = useSelector<RootState, RootState["user"]["error"]>(
```
Comment (Mykyta-Pysarev) [inline src/containers/LoginForm.tsx]:
```suggestion
  const loginError = useSelector<RootState, RootState["user"]["authUser"]["error"]>(
```

File: src/containers/LoginForm.tsx [hunk 17]
```diff
@@ -23,18 +23,13 @@ interface MyForm {
...
     (store) => store.user.error
```
Comment (Mykyta-Pysarev) [inline src/containers/LoginForm.tsx]:
```suggestion
    (store) => store.user.authUser.error
```

File: src/containers/LoginForm.tsx (hunk 16 above)
Comment (Mykyta-Pysarev) [reply src/containers/LoginForm.tsx]: After my commit it will not work. I changed structure of user storage.

**Review by Mykyta-Pysarev — CHANGES_REQUESTED** [review]

**Review by DavidKorystov — APPROVED** [review]

**Review by VitaliiDenysiuk — APPROVED** [review]
lgtm

**Review by Mykyta-Pysarev — APPROVED** [review]

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**